app/views/packages/index.html.erb

```diff
@@ -0,0 +1,94 @@
+<div>
+  <div>
+    <h1>
+      Delivery Tracker
+    </h1>
+    <p>
+      Know if something gets lost in the mail.
+    </p>
+  </div>
+</div>
+
+<hr>
+
+<div>
+  <div>
+    <h2>
+      Expecting a package?
+    </h2>
+
+    <form action="/insert_package" method="post">
+      <div>
+        <label for="description_box">
+          Description
+        </label>
+
+        <input type="text" id="description_box" name="query_description">
+      </div>
+
+      <div>
+        <label for="estimated_arrival_box">
+          Supposed to arrive on
+        </label>
+
+        <input type="date" id="estimated_arrival_box" name="query_estimated_arrival">
+      </div>
+
+      <div>
+        <label for="details_box">
+          Details
+        </label>
+
+        <textarea type="text" id="details_box" name="query_details">
+        </textarea>
+      </div>
+
+      <input type="hidden" name="query_user_id" value="<%= current_user.id %>">
```

Comment:
this is good but could be a security risk, assigning in the controller is best practice :)